### PR TITLE
add more example to the ci test

### PR DESCRIPTION
**Pull request by xiaofengleo**
add examples from https://github.com/Health-RI/starter-kit-info/blob/main/example.ttl to the ci test. These examples were previously used in validator.

### CODE REVIEW

**Comment by konradana** [comment]
Looks good to me

**Review by konradana — APPROVED** [review]

File: Formalisation(shacl)/Core/Example-Data/example-catalog.ttl [hunk 1]
```diff
@@ -1,12 +1,31 @@
 @prefix dcat: <http://www.w3.org/ns/dcat#> .
 @prefix dct: <http://purl.org/dc/terms/> .
 @prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
```
Comment (kburger) [inline Formalisation(shacl)/Core/Example-Data/example-catalog.ttl:4]: The dcterms namespace is already defined on line 2, this is an unnecessary duplicate.

```suggestion
```

File: Formalisation(shacl)/Core/Example-Data/example-catalog.ttl [hunk 2]
```diff
@@ -1,12 +1,31 @@
...
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
```
Comment (kburger) [inline Formalisation(shacl)/Core/Example-Data/example-catalog.ttl:5]: The foaf namespace is already defined on line 3.

```suggestion
```

File: Formalisation(shacl)/Core/Example-Data/example-catalog.ttl [hunk 3]
```diff
@@ -1,12 +1,31 @@
...
+@prefix ns1: <http://www.w3.org/2006/vcard/ns#> .
```
Comment (kburger) [inline Formalisation(shacl)/Core/Example-Data/example-catalog.ttl:6]: The prefix name is not readable.

```suggestion
@prefix vcard: <http://www.w3.org/2006/vcard/ns#> .
```

File: Formalisation(shacl)/Core/Example-Data/example-catalog.ttl [hunk 4]
```diff
@@ -1,12 +1,31 @@
...
+
+
+<http://example.com/catalog/1> a dcat:Catalog ;
+    dcterms:description "Research works of 2nd grade Hogwarts students" ;
+    dcterms:identifier "http://example.com/catalog/1" ;
+    dcterms:issued "1992-01-06T00:00:00"^^xsd:dateTime ;
+    dcterms:title "Student research works 1992" ;dct:publisher [
+        a foaf:Agent ;
+        foaf:name "Test" ;
+		dct:identifier "ID_00001";
```
Comment (kburger) [inline Formalisation(shacl)/Core/Example-Data/example-catalog.ttl:26]: Indentation using tabs instead of spaces, please convert to spaces.

```suggestion
        dct:identifier "ID_00001" ;
```

File: Formalisation(shacl)/Core/Example-Data/example-dataset.ttl [hunk 5]
```diff
@@ -2,6 +2,16 @@
 @prefix dct: <http://purl.org/dc/terms/> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 @prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix dcat: <http://www.w3.org/ns/dcat#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
```
Comment (kburger) [inline Formalisation(shacl)/Core/Example-Data/example-dataset.ttl:7]: Duplicate namespaces.

File: Formalisation(shacl)/Core/Example-Data/example-dataset.ttl [hunk 6]
```diff
@@ -2,6 +2,16 @@
...
+@prefix ns1: <http://www.w3.org/2006/vcard/ns#> .
```
Comment (kburger) [inline Formalisation(shacl)/Core/Example-Data/example-dataset.ttl:8]: Non human-friendly namespace prefix.

File: Formalisation(shacl)/Core/Example-Data/example-dataset.ttl [hunk 7]
```diff
@@ -2,6 +2,16 @@
...
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix vcard: <http://www.w3.org/2006/vcard/ns#> .
```
Comment (kburger) [inline Formalisation(shacl)/Core/Example-Data/example-dataset.ttl:10]: Repeated namespace declarations.

File: Formalisation(shacl)/Core/Example-Data/example-dataset.ttl [hunk 8]
```diff
@@ -2,6 +2,16 @@
...
+@prefix core: <http://coreRule-healthri.nl#> .
+@prefix : <http://coreRule-healthri.nl#> .
```
Comment (kburger) [inline Formalisation(shacl)/Core/Example-Data/example-dataset.ttl:12]: Namespace declarations that are not used in the document.

File: Formalisation(shacl)/Core/Example-Data/example-dataset.ttl [hunk 9]
```diff
@@ -2,6 +2,16 @@
...
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
```
Comment (kburger) [inline Formalisation(shacl)/Core/Example-Data/example-dataset.ttl:13]: Also not used.

**Review by kburger — CHANGES_REQUESTED** [review]

**Review by xiaofengleo — COMMENTED** [review]
looks good

File: Formalisation(shacl)/Core/Example-Data/example-dataset.ttl (hunk 5 above)
Comment (xiaofengleo) [reply Formalisation(shacl)/Core/Example-Data/example-dataset.ttl:7]: resolved in example-dataset.ttl

File: Formalisation(shacl)/Core/Example-Data/example-dataset.ttl (hunk 6 above)
Comment (xiaofengleo) [reply Formalisation(shacl)/Core/Example-Data/example-dataset.ttl:8]: replace ns1 with vcard and deleted ns1 line

File: Formalisation(shacl)/Core/Example-Data/example-dataset.ttl (hunk 8 above)
Comment (xiaofengleo) [reply Formalisation(shacl)/Core/Example-Data/example-dataset.ttl:12]: two lines were removed

File: Formalisation(shacl)/Core/Example-Data/example-dataset.ttl (hunk 9 above)
Comment (xiaofengleo) [reply Formalisation(shacl)/Core/Example-Data/example-dataset.ttl:13]: removed.

File: Formalisation(shacl)/Core/Example-Data/example-catalog.ttl [hunk 10]
```diff
@@ -1,12 +1,29 @@
...
+		dct:identifier "ID_00002";
     ] .
+
+
+<http://example.com/catalog/1> a dcat:Catalog ;
+    dct:description "Research works of 2nd grade Hogwarts students" ;
+    dct:identifier "http://example.com/catalog/1" ;
+    dct:issued "1992-01-06T00:00:00"^^xsd:dateTime ;
+    dct:title "Student research works 1992" ;dct:publisher [
+        a foaf:Agent ;
```
Comment (kburger) [inline Formalisation(shacl)/Core/Example-Data/example-catalog.ttl:22]:
```suggestion
    dct:title "Student research works 1992" ;
    dct:publisher [
        a foaf:Agent ;
```

File: Formalisation(shacl)/Core/Example-Data/example-dataset.ttl [hunk 11]
```diff
@@ -2,6 +2,10 @@
 @prefix dct: <http://purl.org/dc/terms/> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 @prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix dcat: <http://www.w3.org/ns/dcat#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
```
Comment (kburger) [inline Formalisation(shacl)/Core/Example-Data/example-dataset.ttl:6]: Repeated namespace declaration.

```suggestion
```

File: README.md
Comment (kburger) [inline README.md:1]: Why is this file included in this PR?

**Review by kburger — CHANGES_REQUESTED** [review]

File: README.md
Comment (konradana) [reply README.md:1]: It's part of [the same issue](https://health-ri.atlassian.net/browse/TDT-144), the range for `identifier` is updated to Literal in documentation.

File: Formalisation(shacl)/Core/Example-Data/example-dataset.ttl (hunk 11 above)
Comment (xiaofengleo) [reply Formalisation(shacl)/Core/Example-Data/example-dataset.ttl:6]: prefix xsd deleted.

File: Formalisation(shacl)/Core/Example-Data/example-catalog.ttl (hunk 10 above)
Comment (xiaofengleo) [reply Formalisation(shacl)/Core/Example-Data/example-catalog.ttl:22]: done

**Review by xiaofengleo — COMMENTED** [review]
done

**Review by kburger — APPROVED** [review]
